Mathlib/Analysis/SpecialFunctions/ImproperIntegrals.lean

```diff
@@ -51,6 +58,54 @@ theorem integral_exp_neg_Ioi (c : ℝ) : (∫ x : ℝ in Ioi c, exp (-x)) = exp
 theorem integral_exp_neg_Ioi_zero : (∫ x : ℝ in Ioi 0, exp (-x)) = 1 := by
   simpa only [neg_zero, exp_zero] using integral_exp_neg_Ioi 0
 
+theorem integrableOn_exp_mul_complex_Ioi {a : ℂ} (ha : a.re < 0) (c : ℝ) :
+    IntegrableOn (fun x : ℝ => Complex.exp (a * x)) (Ioi c) := by
```

Comment:
You should have remark like the one [here](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Measure/Lebesgue/Integral.html#Substituting-x-for-x) describing why you only have `Ioi` and `Iic` (although, for the `IntegrableOn` statements, as opposed to the integrals themselves, I suspect it's better to have closed infinite intervals everywhere, but I may be wrong.)